mirage/config.js

```diff
@@ -85,6 +85,21 @@ export default function() {
 
   this.delete('/user_categories/:id');
 
+  this.get('/skills');
+  this.get('/user_skills');
+
+  // TODO: Make this work when relationships work
```

Comment:
Which part of this doesn't work? A join model should be fine, since, unlike projects, there isn't a user -> skill many-to-many relationship here, just a user > userSkills, and skill > userSkills.